.github/workflows/t3000-model-perf-tests.yaml

```diff
@@ -17,20 +17,11 @@ jobs:
       fail-fast: false
       matrix:
         test-group: [
-          {
-            name: "T3000 LLM model perf tests",
-            model-type: "LLM",
-            arch: wormhole_b0,
-            runs-on: [arch-wormhole_b0, "config-t3000", "in-service", "runner-test", "bare-metal", "pipeline-perf"],
-            cmd: './tests/scripts/run_tests.sh --tt-arch wormhole_b0 --pipeline-type llm_model_perf_t3000_device --dispatch-mode ""'
-          },
-          {
-            name: "T3000 CNN model perf tests",
-            model-type: "CNN",
-            arch: wormhole_b0,
-            runs-on: [arch-wormhole_b0, "config-t3000", "in-service", "runner-test", "bare-metal", "pipeline-perf"],
-            cmd: './tests/scripts/run_tests.sh --tt-arch wormhole_b0 --pipeline-type cnn_model_perf_t3000_device --dispatch-mode ""'
-          },
+          { name: "t3k LLM falcon7b model perf tests", model-type: "LLM", arch: wormhole_b0, cmd: run_t3000_falcon7b_tests, timeout: 120, owner_id: U052J2QDDKQ}, #Pavle Josipovic
+          { name: "t3k LLM mixtral model perf tests", model-type: "LLM", arch: wormhole_b0, cmd: run_t3000_mixtral_tests, timeout: 120, owner_id: U03PUAKE719}, # Miguel Tairum
+          { name: "t3k LLM llama2 model perf tests", model-type: "LLM", arch: wormhole_b0, cmd: run_t3000_llama2_70b_tests, timeout: 120, owner_id: U03FJB5TM5Y}, #Colman Glagovich
+          { name: "t3k LLM falcon40b model perf tests", model-type: "LLM", arch: wormhole_b0, cmd: run_t3000_falcon40b_tests, timeout: 120, owner_id: U044T8U8DEF}, # Johanna Rock
+          #{ name: "t3k CNN model perf tests ", model-type: "CNN", arch: wormhole_b0, cmd: run_t3000_cnn_tests, timeout: 120, owner_id: }, #No tests are being run?
```

Comment:
Will this fail?

Reply:
if you're referring to the commented out test, 
no its not called at all.

https://github.com/tenstorrent/tt-metal/actions/runs/9670246168

There does seem to be changes I need to do for get this to pass though as its one csv I think